src/app/app.component.scss

```diff
@@ -0,0 +1,7 @@
+.hidden-router {
```

Comment:
What's the purpose of this?

Reply:
This is part of a fix to get sticky footer working and allow the flex container to apply to the router-outlet code.  Without this code, the columns can't stretch to full screen between the header and footer.